code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -68,6 +68,7 @@
 	shoes = /obj/item/clothing/shoes/sneakers/blue
 	l_pocket = /obj/item/pinpointer/crew
 	l_hand = /obj/item/storage/medkit/surgery
+	r_hand = /obj/item/laser_pointer
```

Comment:
put it in their pocket instead

Reply:
ok, one sec

Reply:
(this will save you from editing the unit tests and its a free slot so why put it in hand

Reply:
`r_hand` is where the Plasmamen internals go when they spawn so it should be avoided for a good reason too.

Reply:
done, changed